lms/static/sass/features/_course-experience.scss

```diff
@@ -64,8 +64,8 @@
   }
 }
 
-// Welcome message
-.welcome-message {
+// Welcome message / Latest Update message
+.welcome-message, .update-message{
```

Comment:
@marcotuts I moved the message up, and so now it should be rendering on the same line as the header.